CHANGES.md

```diff
@@ -1,54 +1,75 @@
+# vob 3.0.0 (2021-09-20)
+
+## Breaking changes
+
+* `Vob::from_elem(value: bool, len: usize)` now takes the value first and the
+  number of repetitions of that value second to mirror the most common way this
+  function is defined elsewhere.
```

Comment:
I think it would be better if this was slightly more explicit that we mean `from_elem` as it appears in e.g. `Vec`.

Reply:
Weirdly, `Vec` does not define `from_elem` -- even though other related types in the Rust repo do!

Reply:
oh derp. disregard this then.